packages/packages/api/src/devices/models/nutrition.ts

```diff
@@ -1,9 +1,10 @@
 import { Aminos } from "./common/aminos";
 import { Macros } from "./common/macros";
 import { Micros } from "./common/micros";
+import { FoodItem } from "./common/food-item";
 import { MetriportData } from "./metriport-data";
 
 export interface Nutrition extends MetriportData {
   summary?: { macros?: Macros; micros?: Micros; aminos?: Aminos };
-  // todo: meals/food?
+  foods?: FoodItem[];
```

Comment:
hypernit: `Food` is more brief than `FoodItem` and will also get the point across.

Reply:
Done!